pkg/cmd/pr/checks/checks_test.go

```diff
@@ -198,14 +207,18 @@ func Test_checksRun(t *testing.T) {
 			}
 
 			err := checksRun(opts)
+			if err != nil && tt.throw {
+				assert.Equal(t, tt.wantOut, err.Error())
+				return
+			}
+
```

Comment:
If we make the naming change above this could be changed to 
```go
if tt.wantErr {
  assert.Equal(t, tt.wantOut, err.Error())
} else if tt.nonSuccessfulChecks {
  assert.Equal(t, "SilentError", err.Error())
} else {
  assert.NoError(t, err)
} 
```
This seems a bit more clear to me.